pom.xml

```
@@ -1416,6 +1416,14 @@
                     <groupId>pl.project13.maven</groupId>
                     <artifactId>git-commit-id-plugin</artifactId>
                     <version>4.0.0</version>
+                    <configuration>
+                        <!-- Include only required properties to speed up build (https://github.com/git-commit-id/git-commit-id-maven-plugin/issues/462) -->
+                        <includeOnlyProperties>
+                            <includeOnlyProperty>^git.build.time$</includeOnlyProperty>
```

Since this is a regex, you could want to escape the dot.
You can use `\Q` instead.

```suggestion
                            <includeOnlyProperty>\Qgit.build.time</includeOnlyProperty>
```

---

Thanks. Never knew about `\Q` until now.